scripts/generate-wasm.sh

```diff
@@ -90,3 +93,24 @@ if [[ ${CHECK_DIFF} == 1 ]]; then
     echo "wasm files are up to dated"
   fi
 fi
+
+echo "Destination bucket: ${DST_BUCKET}"
+if [ -n "${DST_BUCKET}" ]; then
+  cd ${ROOT}
+  # Get SHA of proxy repo
+  SHA="$(git rev-parse --verify HEAD)"
+  TMP_WASM=$(mktemp -d -t wasm-plugins-XXXXXXXXXX)
+  trap "rm -rf ${TMP_WASM}" EXIT
+  for i in `find . -name "*.wasm" -type f`; do
+    # Get name of the plugin
+    PLUGIN_NAME=$(basename $(dirname ${i}))
+    # Rename the plugin file and generate sha256 for it
+    WASM_NAME="${TMP_WASM}/${PLUGIN_NAME}-${SHA}.wasm"
+    SHA256_NAME="${WASM_NAME}.sha256"
+    cp ${i} ${WASM_NAME}
+    sha256sum "${WASM_NAME}" > "${SHA256_NAME}"
+    
+    # push wasm files and sha to the given bucket
+    gsutil cp "${WASM_NAME}" "${SHA256_NAME}" "${DST_BUCKET}"
```

Comment:
Do we want to overwrite existing files? If not, then you should verify that the target doesn't already exist (`gsutil stat "${DST_BUCKET}/${WASM_NAME}"`).

Reply:
Done.